eval.go

```diff
@@ -48,6 +48,10 @@ func (e *setExpr) eval(app *app, args []string) {
 		app.ui.loadFile(app, true)
 	case "dirpreviews":
 		gOpts.dirpreviews = true
+	case "nodirpreviews":
+		gOpts.dirpreviews = false
+	case "dirpreviews!":
+		gOpts.dirpreviews = false
```

Comment:
This should say `gOpts.dirpreviews = !gOpts.dirpreviews`.

Reply:
Indeed, thank you for noticing. I wonder how I let that slip through...